### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by akuckartz**
Add table with implementations

### CODE REVIEW

**Comment by pchampin** [comment]
Thanks, that's a very good idea.

I would remove N3 from the list, though, as it is not an *implementation*. [EYE] however, is an implementation of N3 that supports Turtle* (in SA mode).

[EYE]: http://eulersharp.sourceforge.net/

**Comment by pchampin** [comment]
Marked as non substantive for IPR from ash-nazg.

**Comment by hartig** [comment]
@pchampin what does your last comment mean?

@akuckartz thanks for starting such a list here

**Comment by pchampin** [comment]
@hartig my last comment was generated through [a W3C service](https://labs.w3.org/repo-manager/pr/id/w3c/rdf-star/21) that performs some checks on each pull requests.
As I understand: CG reports are subject to IPR agreements, so only pull requests from "registered" contributors should be merged into the report -- unless the content is editorial i.e. "non substansive".

**Comment by akuckartz** [comment]
I am not yet a "registered" member of the CG. But I agree with the IPR agreement.

**Comment by pchampin** [comment]
@akuckartz I think the way to formalize that is to
* join the [RDF-DEV community group](https://www.w3.org/community/rdf-dev/) (which will require you to request  a W3C account), and then to
* [link your github account to your W3C account](https://www.w3.org/users/myprofile/connectedaccounts).

**Comment by pchampin** [comment]
Note that it is not required for this particular PR, as it does not touch the report itself. Hence my marking it as "non substantive".

**Comment by hartig** [comment]
@pchampin thanks for the clarification! So, we can merge this PR now, correct?

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+## Preliminary implementations ##
+
+| SA | PG | Implementation | Notes - Documentation
+|----|----|----------------|----------------------
+|    |  + | AllegroGraph   | in the works - https://lists.w3.org/Archives/Public/public-rdf-star/2020Aug/0021.html
+|    |  x | AnzoGraph      | https://docs.cambridgesemantics.com/anzograph/v2.2/userdoc/lpgs.htm?Highlight=rdf
+|    |  x | BlazeGraph     | https://github.com/blazegraph/database/wiki/Reification_Done_Right
+|  x |    | GraphDB        | http://graphdb.ontotext.com/documentation/9.2/free/devhub/rdf-sparql-star.html
+|  x |    | Jena           | https://jena.apache.org/documentation/rdfstar/
+|  + |  + | N3             | deferred - https://github.com/w3c/N3/issues/27#issuecomment-644768502
```
Comment (pchampin) [inline README.md:14]:
```suggestion
|  x |  + | EYE             | PG can be emulated by adding rules - https://github.com/josd/eye/
```
As mentioned in the discussion, N3 is not an implementation, it is another extension of RDF. EYE is an implemetation of N3 that now also implements RDF* -- AFAIK, RubyRDF also implements both.

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+|  x |    | rdf4j          | https://rdf4j.org/documentation/programming/rdfstar/
+|  x |  + | rdfjs/N3.js    | PG may become configurable soon - https://github.com/rdfjs/data-model-spec/pull/165
+|  x |  x | RubyRDF        | http://rdf.greggkellogg.net/yard/file.rdf-README.html#rdf-rdfstar
+|    |  x | Stardog        | https://www.stardog.com/docs/#_edge_properties
```
Comment (pchampin) [inline README.md:18]:
```suggestion
|    |  x | Stardog        | https://www.stardog.com/docs/#_edge_properties
|    |  x | Corese         | http://ns.inria.fr/sparql-extension/rdfstar.html

**Comment by pchampin** [comment]
> So, we can merge this PR now, correct?

@olaf correct! :)
@akuckartz would you mind applying the suggested changes first? thx

File: README.md [hunk 3]
```diff
@@ -1 +1,21 @@
...
+|----|----|----------------|----------------------
+|    |  + | AllegroGraph   | in the works - https://lists.w3.org/Archives/Public/public-rdf-star/2020Aug/0021.html
+|    |  x | AnzoGraph      | https://docs.cambridgesemantics.com/anzograph/v2.2/userdoc/lpgs.htm?Highlight=rdf
+|    |  x | BlazeGraph     | https://github.com/blazegraph/database/wiki/Reification_Done_Right
+|  x |    | GraphDB        | http://graphdb.ontotext.com/documentation/9.2/free/devhub/rdf-sparql-star.html
+|  x |    | Jena           | https://jena.apache.org/documentation/rdfstar/
+|  x |  + | EYE            | PG can be emulated by adding rules - https://github.com/josd/eye/
+|  x |    | rdf4j          | https://rdf4j.org/documentation/programming/rdfstar/
+|  x |  + | rdfjs/N3.js    | PG may become configurable soon - https://github.com/rdfjs/data-model-spec/pull/165
+|  x |  x | RubyRDF        | http://rdf.greggkellogg.net/yard/file.rdf-README.html#rdf-rdfstar
```
Comment (gkellogg) [inline README.md:17]:
```suggestion
|  x |  x | RubyRDF        | https://github.com/ruby-rdf/rdf#rdf-rdfstar
```

**Pull request merged**